data/helpers.d/setting

```diff
@@ -158,12 +158,21 @@ ynh_webpath_register () {
 
 # Create a new permission for the app
 #
-# example: ynh_permission_create --permission admin --url /admin --allowed alice bob
+# example: ynh_permission_create --permission admin --url /admin --additional_urls 'domain.tld/otherurl /superadmin' --allowed alice bob --label 'My app admin'
 #
-# usage: ynh_permission_create --permission "permission" [--url "url"] [--allowed group1 group2]
+# usage: ynh_permission_create --permission "permission" [--url "url"] [--additional_urls "second-url" [ "other-url" ]] [--auth_header true|false]
+#                                                        [--allowed group1 [ group2 ]] [--label "label"] [--show_tile true|false]
+#                                                        [--protected true|false]
 # | arg: permission - the name for the permission (by default a permission named "main" already exist)
 # | arg: url - (optional) URL for which access will be allowed/forbidden
+# | arg: additional_urls - (optional) List of additional URL for which access will be allowed/forbidden
+# | arg: auth_header - (optional) Define for the URL of this permission, if SSOwat pass the authentication header to the application. Default is true
 # | arg: allowed - (optional) A list of group/user to allow for the permission
+# | arg: label - (optional) Define a name for the permission. This label will be shown on the SSO and in the admin. Default is "APP_LABEL (permission name)".
+# | arg: show_tile - (optional) Define if a tile will be shown in the SSO
+# | arg: protected - (optional) Define if this permission is protected. If it is protected the administrator
+# |                             won't be able to add or remove the visitors group of this permission.
+# |                             By default it's 'true' (for the permission different than 'main').
```

Comment:
Considering a PR is ongoing to standardize the helpers https://github.com/YunoHost/yunohost/pull/947
Would be better to clean that one before merging it.